stats.go

```diff
@@ -22,6 +22,7 @@ import (
 type Stats struct {
 	producerUnflushed uint64
 	producerErr       uint64
+	unknownEventErr   uint64
```

Comment:
Since these are not errors per-se and also for consistency with the other metrics, perhaps this should be named instead: `producerUnknownEvents`.